changedetectionio/forms.py

```diff
@@ -328,6 +328,8 @@ class watchForm(commonSettingsForm):
     ignore_status_codes = BooleanField('Ignore Status Codes (process non-2xx status codes as normal)', default=False)
     trigger_text = StringListField('Trigger/wait for text', [validators.Optional(), ValidateListRegex()])
 
+    share_filters_across_tags = BooleanField('Copy filter/trigger settings to all watches with the same tags')
```

Comment:
Maybe `sync_filters_across_tags` is better `'Copy filter/trigger settings to all watches with the same tags'` also fine

Reply:
Will change that!